framework/tests/value/ValueCast.java

```diff
@@ -0,0 +1,11 @@
+// Test case for issue 1264: https://github.com/typetools/checker-framework/issues/1264
```

Comment:
I actually just went ahead and fixed #1299 and opened a PR #1300. We shouldn't merge this until that's in.